### PR TITLE
fix env file name

### PR DESCRIPTION
**Pull request by samuelarogbonlo**
**Summary of changes**
This change is to avoid mistakenly sending out .env files to the public. 

Changes introduced in this pull request:
- Change of file names with necessary  content 
- ReadMe updated on instructions as to how to successfully test locally with necessary env variables
- Future `.env` is gitignored

### CODE REVIEW

**Comment by lemmih** [comment]
Sounds good. You're missing a few end-of-file newlines and maybe we could name the file `dot-env-template`.

**Comment by samuelarogbonlo** [comment]
> Sounds good. You're missing a few end-of-file newlines and maybe we could name the file `dot-env-template`.

Yes, I just made the changes.

**Comment by LesnyRumcajs** [comment]
@samuelarogbonlo please rebase once https://github.com/ChainSafe/forest-iac/pull/23 is merged so the CI is green

File: scripts/daily_snapshot/README.md [hunk 1]
```diff
@@ -18,3 +18,4 @@ Fedora Linux 36 (Cloud Edition) x86_64
 ## Installation
 * Put Digital Ocean Spaces password and slack api token in `.env`
 * Run `./boot-service`
+* To use the enviromental variables, create a new file called `.env` before adding the credentials for testing. Get the credentials from the content of the `env-develop` file.
```
Comment (LesnyRumcajs) [inline scripts/daily_snapshot/README.md:21]:
```suggestion
* To use the enviromental variables, create a new file called `.env` before adding the credentials for testing. Get the credentials from the content of the `dot-env-template` file. 
```

**Pull request merged**